volume-providers/softlayer/backend/network_storage_service.go

```diff
@@ -124,3 +126,25 @@ func (ns *NetworkStorageServiceSL) AllowAccessFromIPAddressList(ipAddressObjectT
 	})
 	return dtStatus, dtError
 }
+
+//RemoveAccessFromSubnetList removes access from subnet List
+func (ns *NetworkStorageServiceSL) RemoveAccessFromSubnetList(subnetObjectTemplates []datatypes.Network_Subnet) (bool, error) {
+	var bStatus bool
+	var dtError error
+	dtError = retry(func() error {
+		bStatus, dtError = ns.networkStorageService.RemoveAccessFromSubnetList(subnetObjectTemplates)
```

Comment:
is "idem-potency" check required ?  e.g if a volume is not authorized and a de-auth call for subnet/ipaddress comes..will it keep on failing/error ? or should pass ?   if we want to pass, may be we need to check for the Response error code or error message keys.

Reply:
I think you have a valid point here, and we should probably handle errors rather then retry endlessly.
Unfortunately I cant seem to find much documentation on the call itself for what it may return as an error to know if all errors are fatal or if some are retryable.